rust/Cargo.lock

```diff
@@ -1773,8 +1773,8 @@ checksum = "57c0d7b74b563b49d38dae00a0c37d4d6de9b432382b2892f0574ddcae73fd0a"
 
 [[package]]
 name = "payjoin"
-version = "0.18.0"
-source = "git+https://github.com/LtbLightning/rust-payjoin?branch=clone-pj-uri-builder#3b5bda95fc8ce63f506538f0ff9daed885810488"
+version = "0.19.0"
+source = "git+https://github.com/payjoin/rust-payjoin?rev=941a6798f52f60d72061fc0a02b5b42146321453#941a6798f52f60d72061fc0a02b5b42146321453"
```

Comment:
@kumulynja as payjoin has been updated with your latest changes. Pleas update the new commit hash. or branch.

Reply:
Updated 👍